hawkular-nest/hawkular-nest-itest/pom.xml

```diff
@@ -290,23 +294,34 @@
       </plugin>
 
       <plugin>
-        <artifactId>maven-resources-plugin</artifactId>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
         <executions>
           <execution>
-            <id>copy-resources</id>
+            <id>create-itest-users</id>
```

Comment:
> Why do you have a random password for the test user?

As I said, first, to ensure that we do not rely on hard coded defaults too much and second to make it possible to run the itests from cmdgw also in Agent safely (where the passwords may be different).

To put it on other words, with the static `*.properties` files, one stores the same piece of info (username and password) on at least two places: (1) encoded in the `*.properties` file and plain either in pom or in the tests. With the present implementation there is just a single place where the username and password pair is stored: in the `pom.xml` file and it is distributed everywhere where it is needed from there.

And having random passwords makes the test system less vulnerable :)